test/client/fraudnet.js

```diff
@@ -0,0 +1,21 @@
+/* @flow */
+import { loadFraudnet } from "../../src/fraudnet";
+
+describe("fraudnet.js", () => {
```

Comment:
So much of this in service of testing fraudnet lol...

TODO: test fraudnet uh duhhh